community/detectors/gradio_cve_2023_51449/src/main/java/com/google/tsunami/plugins/detectors/cves/cve202351449/Cve202351449VulnDetector.java

```diff
@@ -156,12 +160,12 @@ private HttpResponse sendGetFileRequest(NetworkService networkService, String pa
             .build());
   }
 
-  private boolean isServiceVulnerable(NetworkService networkService) {
+  private DetectionResult checkIfServiceVulnerable(NetworkService networkService) {
```

Comment:
Consider renaming it to `toDetectionResult`

Reply:
@vsutedjo done